src/super_gradients/training/metrics/segmentation_metrics.py

```diff
@@ -101,26 +167,73 @@ def compute(self):
         return pix_acc
 
 
-class IoU(torchmetrics.IoU):
-    def __init__(self, num_classes, dist_sync_on_step=True, ignore_index=None):
-        super().__init__(num_classes=num_classes, dist_sync_on_step=dist_sync_on_step, ignore_index=ignore_index)
+class IoU(torchmetrics.JaccardIndex):
```

Comment:
Notice we are currently using 0.7.3, did you check inheriting from torchmetrics.JaccardIndex works with that version ?

Reply:
Yes, IoU was renamed JaccardIndex at version `0.7.0`, see [here](https://github.com/PyTorchLightning/metrics/pull/662)